cpp/include/cudf/strings/detail/convert/fixed_point.cuh

```diff
@@ -52,7 +52,13 @@ __device__ inline thrust::pair<UnsignedDecimalType, int32_t> parse_integer(
 
   while (iter < iter_end) {
     auto const ch = *iter++;
-    if (ch == decimal_pt_char && !decimal_found) {
+    if (ch == decimal_pt_char) {
```

Comment:
Why do we need this change? The data is incorrect. Checking the format of the data should be handled by `is_fixed_point` not here.

Reply:
Bobby is right. This if-check should not go here.